app/src/main/java/com/parasoft/demoapp/util/TimeUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.parasoft.demoapp.util;
+
+import android.util.Log;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class TimeUtil {
+    private static final String TAG = "TimeUtil";
+
+    public static String convertToLocalTime(String time) {
```

Comment:
Does this method work on your side? For me it's still displaying the UTC time.
In my case I'll need to see PT time and you'll need to CST time.
Please try the following:
1. Remove "Locale.CHINA" in line 16. (Let me know if there's reason for that but it doesn't look like)
2. Remove line 17 and add "simpleDateFormat.setTimeZone(TimeZone.getDefault());" at line 20.
3. Return formatted date inside try block and return null outside.

Reply:
To make the method works, I think you need to check and change the timezone on your android emulator, it will get timezone from your android deivice.

Reply:
Please try to change your time zone to PT to see whether it's displaying correct time.

Reply:
I add "@SuppressLint("SimpleDateFormat")" to ignore a warning about "new SimpleDateFormat". This may be helpful: https://stackoverflow.com/questions/17807777/simpledateformatstring-template-locale-locale-with-for-example-locale-us-for